frontend/src/pages/Administration.vue

```diff
@@ -135,22 +135,23 @@ export default {
     ]),
     ...mapGetters([
       'shootList',
-      'projectList'
+      'projectList',
+      'canPatchProject',
+      'canDeleteProject',
+      'projectFromProjectList',
+      'costObjectSettings'
     ]),
     project () {
-      return projectFromProjectList()
+      return this.projectFromProjectList
     },
     projectDetails () {
       return getProjectDetails(this.project)
     },
-    costObjectSettings () {
-      return getCostObjectSettings() || {}
-    },
     costObjectSettingEnabled () {
-      return getCostObjectSettings() !== undefined
+      return this.costObjectSettings !== undefined
```

Comment:
``js
return !this.costObjectSettings
``

Reply:
will change it to !isEmpty(this.costObjectSettings) as discussed